pkg/pillar/Makefile

```diff
@@ -32,11 +32,19 @@ $(DISTDIR):
 
 build: $(APPS) $(APPS1)
 
+TAGS=
+ifeq ($(RSTATS),y)
+	TAGS:=-tags rstats 
```

Comment:
yetus adds github annotations which you can see in the diffs (after you see that the yetus run failed). Makes sense to fix the two blanks complaints.

Reply:
fixed.